docs/using/php.rst

```diff
@@ -33,7 +33,7 @@ very creative, let's call it "blog":
 
 ::
 
-    $ tsuru app create blog php
+    $ tsuru app create blog php --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team